string seqId
int64 actionTime 
string deviceId
string dbm
string electricQuantity
string status
string outdoorX
string outdoorY
string indoorX
string indoorY
string floorNo
string exceptionDetail

